Gemfile

```diff
@@ -1,30 +1,32 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
-ruby "3.3.0"
+ruby '3.3.0'
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
-gem "rails", "~> 7.1.3"
+gem 'rails', '~> 7.1.3'
+
+gem 'rubocop', '>= 1.0', '< 2.0'
 
 # The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
-gem "sprockets-rails"
+gem 'sprockets-rails'
 
 # Use sqlite3 as the database for Active Record
-gem "sqlite3", "~> 1.4"
+gem 'sqlite3', '~> 1.4'
 
```

Comment:
- [ ] As stated in the requirement, knidly ensure to use postgres as your datase. You will need to add it to the gem file and change the config settings in /config/database.yml file. 🙏 

**Note: Kindly ensure that your app runn smoothly before you re-submit**